scale/ingest/models.py

```diff
@@ -64,6 +68,35 @@ def __init__(self, strike, most_recent=None, files=0, size=0, values=None):
 class IngestManager(models.Manager):
     """Provides additional methods for handling ingests."""
 
+    def create_ingest(self, file_name, workspace, scan_id=None, strike_id=None):
+        """Creates a new ingest for the given file name. The database save is the caller's responsibility.
+
+        :param file_name: The name of the file being ingested
+        :type file_name: string
+        :param workspace:
+        :type workspace: string
+        :param scan_id:
+        :type scan_id: int
+        :param strike_id:
+        :type strike_id: int
+        :returns: The new ingest model
+        :rtype: :class:`ingest.models.Ingest`
+        """
+
+        ingest = Ingest()
+
+        if scan_id:
+            ingest.scan_id = scan_id
+        if strike_id:
+            ingest.strike_id = strike_id
+
+        ingest.file_name = file_name
+        ingest.media_type = get_media_type(ingest.file_name)
+        ingest.workspace = workspace
+
+        logger.info('New file on %s: %s', ingest.workspace.name, ingest.file_name)
```

Comment:
I would recommend taking out the log statement here, as it seems a little low level. Logging in strike and scan would have more context.